account-kit/signer/src/base.ts

```diff
@@ -604,6 +641,30 @@ export abstract class BaseAlchemySigner<TClient extends BaseSignerClient>
     return user;
   };
 
+  private authenticateWithOauth = async (
+    args: Extract<AuthParams, { type: "oauth" }>
+  ): Promise<User> => {
+    const params: OauthParams = {
+      ...args,
+      expirationSeconds: this.sessionManager.expirationTimeMs,
```

Comment:
does this need to be 
```suggestion
      expirationSeconds: Math.floor(this.sessionManager.expirationTimeMs / 1000),
```